app/models/google_calendar/event_wrapper/event.rb

```diff
@@ -0,0 +1,110 @@
+module GoogleCalendar
```

Comment:
This class looks like it has too many responsibilities. E.g. it shouldn't do nil checks on every parameter it works with. It should be done in some other place, maybe in EventWrapper builder?